Library/Homebrew/dev-cmd/bump-formula-pr.rb

```diff
@@ -345,6 +345,9 @@ def bump_formula_pr
         pr_message = <<~EOS
           Created with `brew bump-formula-pr`.
         EOS
+
+        args.each_pair { |k, v| pr_message += "#{k}: #{v}\n" if k.to_s.scan(/url|version|sha256|tag|revision/) != [] }
```

Comment:
This line is really long; can you adjust this to use a block-form `each_pair` and `next unless` instead of `if`?

Reply:
Fixed.